ooniprobe/View/DashboardTableViewController.m

```diff
@@ -73,7 +102,7 @@ -(IBAction)run:(id)sender{
                                message:NSLocalizedString(@"Modal.Error.NoInternet", nil) inView:self];
 }
 
--(void)runAll{
+-(IBAction)runAll{
```

Comment:
Here you are not returning anything, yet you changed the return value. Why is that?

Reply:
`IBAction` is needed to connect a function in a storyboard object

Reply:
I don't get why you aren't returning anything? Is there a default `return nil`?

Reply:
So, it turns out `IBAction` resolves to `void`